composer.json

```
@@ -16,7 +16,7 @@
         "ext-zlib": "*"
     },
     "require-dev": {
-        "zf1s/phpunit": "^3.7",
+        "zf1s/phpunit": ">=3.7.39",
```

```suggestion
        "zf1s/phpunit": "^3.7.39",
```

lets still keep 3.7 series